drivers/metal/rendering_device_driver_metal.h

```diff
@@ -28,12 +28,9 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
-#ifndef RENDERING_DEVICE_DRIVER_METAL_H
```

Comment:
Missed `#include "drivers/metal/rendering_device_driver_metal.h"` in the `modules/openxr/extensions/platform/openxr_metal_extension.mm`.